app/templates/application.hbs

```diff
@@ -1,13 +1,21 @@
-{{page-title "My"}}
+{{page-title 'My | Real Dev Squad'}}
 
-<Navbar @firstName={{@model.firstName}} @profilePictureURL={{@model.profilePictureURL}} @signOutHandler={{this.signOutHandler}} />
+<Navbar
+  @firstName={{@model.firstName}}
+  @profilePictureURL={{@model.profilePictureURL}}
+  @signOutHandler={{this.signOutHandler}}
+/>
 
-<div class="main-container">
+<div class='main-container'>
```

Comment:
I don't think that this should be changed

Reply:
Rectified it. Was caused due to my local prettier settings.